content/applications/inventory_and_mrp/inventory/management/misc/cluster_picking.rst

```diff
@@ -0,0 +1,221 @@
+Locations` and :guilabel:`Multi-Step Routes` options under the :guilabel:`Warehouse` heading must
+also be checked on this settings page. When finished, click :guilabel:`Save`.
+
+.. image:: cluster_picking/locations-routes-checkbox.png
+   :align: center
+   :alt: Enable *Storage Locations* and *Multi-Step Routes* Inventory > Configuration > Settings.
+
+.. _inventory/misc/create-package:
+
+Packages setup
+--------------
+
+After the :guilabel:`Packages` feature is enabled, set up the packages before the picking, in the
+:menuselection:`Inventory app --> Products --> Packages` page. Click the :guilabel:`New` button to
+create a new package.
+
+On the package form for creating a new package, the :guilabel:`Package Reference` is pre-filled to
+the next available `PACK` number in the system. :guilabel:`Pack Date` is automatically set to the
+creation date of the form.
+
+For the :guilabel:`Package Use` field, the :guilabel:`Disposable Box` radio option should be
+selected if the package is a shipping container. On the contrary, the :guilabel:`Reusable Box`
+option should be selected if the package is simply used as a method of grouping products from the
+same |SO| together and the products are intended to be moved to the intended shipping container at
+the output location.
+
+.. seealso::
+   :ref:`Packages <inventory/management/packages>`
+
+.. example::
+   A package intended for cluster picking is named `CLUSTER-PACK-3` for easy identification. For
+   this workflow, the products are directly packed using their intended shipping containers, so
+   :guilabel:`Package Use` is set to :guilabel:`Disposable Box`.
+
+   .. image:: cluster_picking/cluster-package.png
+      :align: center
+      :alt: Create new package form.
+
+Create cluster batch
+====================
+
+To see how cluster picking works in Odoo, navigate to the *Sales* app and create |SOS| that will be
+fulfilled together in the same batch. After confirming the |SO|, the :guilabel:`Delivery` smart
+button becomes visible. Displayed inside the icon is a number representing the amount of steps in
+the outgoing shipment process.
+
+.. example::
+   Begin by creating three |SOS| for the apples, oranges, and bananas as shown in the :ref:`example
+   above <inventory/misc/cluster_picking/example>`.
+
+   After confirming the |SO|, the :guilabel:`Delivery` smart button will be shown with the number
+   `2`, indicating that there are two operations to complete: `Pick` and `Delivery`.
+
+   .. image:: cluster_picking/create-sales-order.png
+      :align: center
+      :alt: Example sales order for an apple, orange, and banana.
+
+With the |SOS| created, orders now must be grouped into batches. To do so, navigate to the
+*Inventory* dashboard and select the operation type card, `Delivery Orders` or `Pick`. Doing so
+displays a filtered list of outgoing operations with the :guilabel:`Ready` status, indicating that
+all the products in the |SO| are in stock.
+
+.. note::
```

Comment:
I asked Sho if it was possible to create a batch during a 1-step delivery and this is his answer:

> This one is a bit complicated, but yeah that should be possible to have any picking type on one move. The reason we separate the picking from the delivery, is mostly for organization purposes. Functionally/physically (and depending on the WH setup and stuff), the picker might also deliver the good. Maybe its a small place and you only have 1 person who does it all. 

> Separating it (multi step) is more for someone who is bigger or has more staff. They have dedicated pickers and packers/delivererers and want to organize what they pick. If you are picking, you may want to group some picks together based on your layout or the workflow you want your operators to follow. You might just have the one delivery order to go out, but still want the organization so when the operator goes and grabs the stuff, its organized for them. Once they grab it and deliver it, they mark it as done (but it all one one step).

> its more common with 1 step (because you have dedicated pickers who bring the stuff to the delivery area, but its not crazy uncommon to want to have specific batch pickings for only a delivery